css-grid-1/grid-items/grid-item-containing-block-001.xht

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
Nit: We should use HTML for new tests.

There was some discussion on the mailing list about that: https://lists.w3.org/Archives/Public/public-css-testsuite/2015Nov/0000.html

I know most of the tests on the grid layout suite are using XHTML but let's avoid it for new tests. :-)